src/Amazon.AspNetCore.DataProtection.SSM/SSMXmlRepository.cs

```diff
@@ -141,22 +140,49 @@ public void StoreElement(XElement element, string friendlyName)
 
         private async Task StoreElementAsync(XElement element, string friendlyName)
         {
-            var parameterName = _parameterNamePrefix + 
+            var parameterName = _parameterNamePrefix +
                             (friendlyName ??
                             element.Attribute("id")?.Value ??
                             Guid.NewGuid().ToString());
 
+            var tier = ParameterTier.Standard;
+            var keyValue = element.ToString();
+            var keyLength = keyValue.Length;
+            // Check if the value is too big for the advanced tier (8192 characters/ 8KB), in this case the key generation is not suitable for keys that should be stored as SSM parmameter.
+            int advancedTierMaxSize = 8192;
```

Comment:
Add the const keyword to make it a locally constant. Also use `var` to be consistent with all the other variables.